utils/auth.js

```diff
@@ -31,5 +31,7 @@ export const redirectIfNotAuth = async ({ req, res, query }) => {
     }`,
   });
   res.end();
-  return {};
+  return {
+    props: {},
```

Comment:
This fixes an issue I think we had introduced after upgrading Next.js. It used to work without returning an empty `props` key, but now throws an error without it!